testsuite/conftest.py

```diff
@@ -74,3 +78,30 @@ def download_all_submissions_stub(assignment, request, queryset):
         "download_all_submissions",
         download_all_submissions_stub,
     )
+
+
+@pytest.fixture
+def mock_config(monkeypatch):
```

Comment:
Moved this fixture here, since I need it for the TA test.

Reply:
Great :+1:. I forgot about moving it.